### PR TITLE
Slack feedback

### PR DESCRIPTION
**Pull request by VojtechVitek**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,8 @@
 [![GoDoc](https://godoc.org/github.com/goware/disque?status.png)](https://godoc.org/github.com/goware/disque)
 [![Travis](https://travis-ci.org/goware/disque.svg?branch=master)](https://travis-ci.org/goware/disque)
 
+*Note: The below examples ignore error handling for readability.*
```
Comment (Mistobaan) [inline README.md]: *The examples below

File: disque.go [hunk 2]
```diff
@@ -9,17 +9,18 @@ import (
...
+// Pool represent Redis connection to a Disque Pool
+// with a certain Disque configuration.
+type Pool struct {
+	redis *redis.Pool
+	conf  Config
 }
 
-// Connect creates new connection to a given Disque Pool.
-func Connect(address string, extra ...string) (*Conn, error) {
+// New creates new connection to a given Disque Pool.
```
Comment (Mistobaan) [inline disque.go]: creates `a` new connection ..

File: disque.go [hunk 3]
```diff
@@ -9,17 +9,18 @@ import (
...
+func New(address string, extra ...string) (*Pool, error) {
```
Comment (Mistobaan) [inline disque.go:20]: here is where I see the function options being useful.

Comment (VojtechVitek) [inline disque.go:20]: I will think about it, maybe I'll reconsider. It's not quite common method yet. But it's elegant, I agree.

Thanks for all your useful feedback, Fabrizio!

**Pull request merged**